src/yarle.ts

```diff
@@ -32,25 +33,31 @@ export const parseStream = async (options: YarleOptions): Promise<void> => {
 
       return reject();
     };
-  
+
     xml.collect('tag');
     xml.collect('resource');
-    xml.on('startElement: en-export', (enExport: any) => {
-      totalNotes = Array.isArray(enExport.note) ? enExport.note.length : 1;
-    });
 
     xml.on('endElement: note', (note: any) => {
-      processNode(note);
-      ++noteNumber;
-      console.log(`Notes processed: ${noteNumber}`);
+      if (
+          options.skipWebClips &&
+          (
+            note['note-attributes']['source-application'] === 'webclipper.evernote' ||
+            note['note-attributes']['source'] === 'web.clip7'
```

Comment:
Could you please create an util function instead to identify that the particular note is a web.clip, like:

``` 
export const isWebClip= (note): boolean => {
return note['note-attributes'] && (
note['note-attributes']['source-application'] === 'webclipper.evernote' ||
            note['note-attributes']['source'] === 'web.clip7');

}

Reply:
Done.